telegram-translation-shortcuts.user.js

```diff
@@ -70,16 +70,53 @@ function scrollItems (down) {
  * Clicks the 'Add Translation' button
  */
 function addTranslation () {
-  document.getElementsByClassName('key-add-suggestion-header').item(0).click()
+  // don't click if input box is expanded
+  var input_wrapper_collapsed = (document.getElementsByClassName('key-add-suggestion-wrap')[0].className == 'key-add-suggestion-wrap');
+  
+  if (!input_wrapper_collapsed)   // don't toggle 'add' and 'cancel'
+    document.getElementsByClassName('key-add-suggestion-header').item(0).click() 
+  else
+    document.getElementsByClassName('input form-control tr-form-control key-add-suggestion-field').item(0).focus() //fixme: throws undefined error
```

Comment:
Does this actually do anything if it throws undefined?

Reply:
No, not really, idk why it throws undefined, but I feel like it needs fixing xD

Reply:
No, I was asking a different thing. :D If it throws undefined, does it have any effect? Wouldn’t it be the same the line?

Reply:
It doesn't have any effect, afaik.

I made it focus to the form input when you press CTRL+A, if it wasn't already in focus.

cuz sometimes, when you are in the middle of typing something, and you get distracted, and then return back to the browser with a mouse click somewhere that's not in the input box, and then you press CTRL+A, that input will collapse, and whatever you had typed will go away.

So this is just a precaution for using a keyboard shortcut you weren't fully aware of what it did. You can always ESC from the form and CTRL+A, if you wanna edit the default or whatever.